README.md

```diff
@@ -232,7 +235,11 @@ To enable autoscaling, add an autoscaler section similar to the one below to you
     "interval": "2",        # Execution interval of the autoscaler in minutes
     "max": "20",            # Maximum number of workers
     "factor": "0.3",        # Percentage of YARN containers that should be running at any point in time 0.0 to 1.0.
-    "downscale": "false"    # Whether or not to downscale. Highly experimental! Please check our notes on downscaling!
+    "downscale": "false",    # Whether or not to downscale.
+    # If downscale is enabled, how long in minutes to wait for active jobs to finish
+    # before terminating nodes and potentially interrupting those jobs.
+    # Note that the autoscaler will not be able to add nodes during this interval.
```

Comment:
How does this interact with the `"interval": "2"` a few lines up? As there's no lock in place another scale will take place, does `gcloud` prevent scales during `graceful-decommission-timeout`?

Reply:
Not ideal, but the server will reject the concurrent update request -- essentially no more updates will happen graceful-decommission-timeout.

Reply:
That sounds fine.